changelogs/7.16.asciidoc

```diff
@@ -0,0 +1,72 @@
+[[release-notes-7.16]]
+== APM version 7.16
+
+https://github.com/elastic/apm-server/compare/7.15\...7.16[View commits]
+
+* <<release-notes-7.16.0>>
+
+[float]
+[[release-notes-7.16.0]]
+=== APM version 7.16.0
+
+https://github.com/elastic/apm-server/compare/v7.15.2\...v7.16.0[View commits]
+
+[float]
+==== Breaking Changes
+- Removed unused stacktrace/frame monitoring counters {pull}5984[5984]
+- Removed unused support for top-level metricsets and metricset tags for RUMv3 {pull}6065[6065]
+- Removed `apm-server.mode` configuration, and "experimental" fields {pull}6086[6086]
+- `transaction.sampled` is now only set for sampled transactions {pull}6066[6066]
+- Unknown metrics are dropped when `transaction.*` or `span.*` are present in a metricset {pull}6111[6111]
+- Removed `metricset.period` from service_destination metrics {pull}6111[6111]
+- Removed `http.request.socket` fields {pull}6152[6152]
+- Removed unused `transaction.duration.{count,sum.us}` metric fields {pull}6180[6180]
+- Removed `ProcessPending` self-instrumentation events {pull}6243[6243]
+- Removed warm phase from default ILM policy {pull}6322[6322]
+- Removed unused `transaction.breakdown.count` metric field {pull}6366[6366]
+- experimental:["This breaking change applies to the experimental tail-based sampling feature."] Changed `apm-server.sampling.tail.events.*` metrics semantics {pull}6273[6273]
+- experimental:["This breaking change applies to the experimental tail-based sampling feature."] Removed `apm-server.sampling.tail.storage_dir` config {pull}6236[6236]
+
+[float]
+==== Bug fixes
+- Use timestamp of original events for transaction/span metrics {pull}6311[6311]
+- Populate the timestamp field for `agent_config` metricsets {pull}6382[6382]
+- Query elasticsearch output `cluster_uuid` on startup {pull}6591[6591]
+- Skip unhandled fields in bulk responses in the experimental Elasticsearch output {pull}6631[6631]
+- Fix the format of `@timestamp` to include fractional seconds when using experimental Elasticsearch output {pull}6646[6646]
+- NaN and +/-Inf values are now filtered out when translating OTLP metrics {pull}6698[6698]
+- Agent configuration metrics are now sent to the metrics-apm.internal* data stream {pull}6767[6767]
+- Improve logged message when APM Server blocks ingestion to wait for the integration to be installed {pull}6796[6796]
```

Comment:
The last two entries didn't make it for `7.16.0`, but they'll be in `7.16.1`. According to: https://github.com/elastic/apm-server/commits/fbec762a40361b683cce466784fda7d9080f13e8.

Reply:
Thanks! Fixed in https://github.com/elastic/apm-server/pull/6825.